### DISCUSSION TITLE
Revert whole task

### DISCUSSION THREAD

**Discussion by BradPerbs** [cline/cline#757] [Feature Requests]
Would be extremely helpful to have a full Revert button to revert all changes inside a task to the starting code.

This is when the AI fk up the code, it's basically impossible to roll back with ctrl z

At the moment I'm forced to save a copy of the files before each task.

**Comment by yisrael-rosen** [comment]
Use git
https://github.com/cline/cline/issues/177#issuecomment-2325121693
